components/CaseStatus/CaseStatusDetailsTable.spec.tsx

```diff
@@ -44,6 +44,23 @@ describe('CaseStatusDetailsTable component', () => {
     expect(elements).not.toBeNull();
     expect(queryByText('09 Sept 2021')).toBeInTheDocument;
   });
+  it('displays the end date, if there is no answers', async () => {
+    const { getByTestId, queryByText } = render(
+      <CaseStatusDetailsTable
+        status={mockedCaseStatusFactory.build({
+          id: 1,
+          type: 'CIN',
+          startDate: '2021-09-09',
+          endDate: '2022-09-09',
+          answers: [],
+        })}
+      />
+    );
+
+    const elements = getByTestId('start_end_date');
+    expect(elements).not.toBeNull();
+    expect(queryByText('09 Sept 2022')).toBeInTheDocument;
```

Comment:
Does this need to check for 'Sep' or 'Sept'?

Reply:
Hi Rachel, it depends if this is ran on a Unix or a Windows machine :) 

I've made them for Unix so this should pass (let me know in case it doesn't on your environment!)